src/NUnitFramework/framework/Constraints/ExactCountConstraint.cs

```diff
@@ -43,12 +53,10 @@ public class ExactCountConstraint : PrefixConstraint
         public ExactCountConstraint(int expectedCount, IConstraint itemConstraint)
             : base(itemConstraint)
         {
-            this.expectedCount = expectedCount;
-            this.DescriptionPrefix = expectedCount == 0
-                ? "no item"
-                : expectedCount == 1
-                    ? "exactly one item"
-                    : string.Format("exactly {0} items", expectedCount);
+            Guard.ArgumentNotNull(itemConstraint, "itemConstraint");
```

Comment:
Well, I was looking for a core team member to bite. :-)

I didn't expect it to go to the public list but it's actually a good idea to get feedback before deciding. Note that the nunit-developer list is for anyone who is __interested in__ NUnit development and has quite a few members.

At the moment you can only contact all the core team members by typing their names. I suggested setting up a list of some kind but somebody thought we had too many already. However, since there are only five of us, you only have to type four names so it's not so bad.

Now that we are between releases for a day or two I'll try to get out my draft of the governance document for review.